agents/PCS-Large-LP-Deposit-Withdrawal/src/agent.spec.ts

```diff
@@ -0,0 +1,298 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent } from "forta-agent";
```

Comment:
Would be better to create a `test` folder and move `agent.spec.ts` `pool.fetcher.spec.ts` test suites to test.

Reply:
Again this is a convention all bots deployed on Forta seem to be following and all the previous bots follow too.